packages/ui/__stories__/Button.stories.tsx

```diff
@@ -156,3 +156,53 @@ export const LinkSemantics = () => {
     </>
   )
 }
+
+export const AllVariants = () => {
+  const props: ButtonProps<ButtonProps> = {
+    className: storyStyles.button,
+    children: 'Button',
+  }
+  return (
+    <>
+      <div className={storyStyles.row}>
+        <Button {...props} variant="contained" color="primary" />
+        <Button {...props} iconLeft={Info} iconLeftAriaLabel={'Additional information'} variant="contained" color="primary" />
+        <Button {...props} variant="contained" color="secondary" />
+        <Button {...props} iconLeft={Info} iconLeftAriaLabel={'Additional information'} variant="contained" color="secondary" />
+
+        <Button {...props} variant="outlined" color="primary" />
+        <Button {...props} iconLeft={Info} iconLeftAriaLabel={'Additional information'} variant="outlined" color="primary" />
+        <Button {...props} variant="outlined" color="secondary" />
+        <Button {...props} iconLeft={Info} iconLeftAriaLabel={'Additional information'} variant="outlined" color="secondary" />
+
+        <Button {...props} variant="text" color="primary" />
+        <Button {...props} iconLeft={Info} iconLeftAriaLabel={'Additional information'} variant="text" color="primary" />
+        <Button {...props} variant="text" color="secondary" />
+        <Button {...props} iconLeft={Info} iconLeftAriaLabel={'Additional information'} variant="text" color="secondary" />
```

Comment:
Is there a difference between `variant="text" color="primary"` and `variant="text" color="secondary"`?

Reply:
Text inherits color so there is no deference. Maybe we will need to change it